homeassistant/components/sensor/zabbix_triggers.py

```diff
@@ -0,0 +1,154 @@
+"""
```

Comment:
Hi @balloob.  Thanks for the review, will fix up all identified errors.  

For this comment, since there will be additional types of Zabbix sensor in the future (Zabbix Item Value) for example, wouldn't we need to separate these out into individual python files?  Also, doesn't the filename need to match the 'platform' type in the configuration file?